assets/js/state/selectors/checksResultsFilters.js

```diff
@@ -0,0 +1,4 @@
+export const getSelectedFilters =
+  (clusterID) =>
```

Comment:
I would change `clusterID` by `groupID/resourceID` once we started making it generic